.github/workflows/push-dev-environment.yaml

```diff
@@ -1,23 +1,134 @@
-name: Push code to dev environment
+name: Build images from latest dev branch
 
 on:
   push:
-    branches: [dev]
+    branches:
+      - dev
 
 jobs:
-  build:
+  create-dev-image:
+    name:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - name: setup-docker
+      - name: Checkout Repo
+        uses: actions/checkout@v1
+
+      - name: Setup Docker
         uses: docker-practice/actions-setup-docker@0.0.1
-      - name: run push
+
+      - name: Log in to docker registry
+        run: |
+          echo ${{ secrets.GITHUB_TOKEN }} | docker login \
+            docker.pkg.github.com \
+            --username ${{ github.actor }} \
+            --password-stdin
+
+      - name: Build new images and push
+        run: |
+          make \
+            build-all \
+            push-images
+        env:
+          # webapp
+          PORT: "3000"
+          REACT_APP_TTILE: "EOS Jungle Testnet Network Dashboard"
+          REACT_APP_DEFAULT_PRODUCER_LOGO: "https://bloks.io/img/eosio.png"
+          REACT_APP_EOS_RATE_LINK: "https://jungle.eosrate.io:8080"
+          REACT_APP_USE_REWARDS: "true"
+          REACT_APP_USE_VOTES: "true"
+          REACT_APP_USE_BLOCK_PRODUCER_AGREEMENT_CONTRACT: "true"
+          REACT_APP_HASURA_URL: "https://graphql-testnet.eosio.cr:3000/v1/graphql"
+          REACT_APP_EOS_API_NETWORK_NAME: "jungle"
+          REACT_APP_EOS_API_HOST: "jungle.eosio.cr"
+          REACT_APP_EOS_API_PORT: "443"
+          REACT_APP_EOS_API_PROTOCOL: "https"
+          REACT_APP_EOS_CHAIN_ID: "2a02a0053e5a8cf73a56ba0fda11e4d92e0238a4a2aa74fccf46d5a910746840"
+          REACT_APP_EOS_DEFAULT_EXCHANGE_RATE: "2.49"
+          REACT_APP_EOS_DEFAULT_EXCHANGE_RATE_API: ${{ secrets.REACT_APP_EOS_DEFAULT_EXCHANGE_RATE_API }}
+          REACT_APP_EOS_USE_BP_JSON_ON_CHAIN: "false"
+          REACT_APP_EOS_BP_JSON_ON_CHAIN_CONTRACT: "producerjson"
+          REACT_APP_EOS_BP_JSON_ON_CHAIN_TABLE: "producerjson"
+          REACT_APP_EOS_BP_JSON_ON_CHAIN_SCOPE: "producerjson"
+          REACT_APP_EOS_API_NETWORK_LABEL: "Jungle"
+          REACT_APP_EOS_API_NETWORK_LOGO: "https://bloks.io/img/eosio.png"
+
+      - name: Build kubernetes files
+        id: build_kubernetes_files
+        run: |
+          make \
+            build-kubernetes
+        env:
+          # general
+          NAMESPACE: jungle
```

Comment:
we already have a namespace called `jungle` for the EOSIO nodes 

lets make a new namespace for the monitor , maybe something like `jungle-dashboard` or `jungle-monitor` ?